shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Value.hs

```diff
@@ -0,0 +1,258 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -Wno-unused-binds #-}
+{-# OPTIONS_GHC -fno-warn-missing-signatures #-}
+
+module Cardano.Ledger.Value where
+
+{-
```

Comment:
I suggest we remove these or alternatively, add as exports

Reply:
I will clean this up. I wanted to get some comments, and it is much easier to debug when we export everything.